tests/test_generateBuildrootSBOM.py

```diff
@@ -0,0 +1,5 @@
+def test_create_buildroot_sbom():
+    import generateBuildrootSBOM
```

Comment:
:x: this import is wrong. it imports from global side-packages
instead, use relative import
```py
import ..generateBuildrootSBOM
```

Reply:
Thanks, that syntax is new to me.

Reply:
since you've added the `__init__.py` files every folder became a package.
therefore you can use [Package Relative Imports](https://docs.python.org/3/reference)/import.html#package-relative-imports